bab/3-desain-implementasi.tex

```diff
@@ -11,7 +11,7 @@ \section{Deskripsi Sistem}
 Sistem akan dibuat dengan \lipsum[1-2]
 
 \section{Implementasi Alat
-\label{sec:implementasi alat}}
+  \label{sec:implementasi alat}}
```

Comment:
Any reason to add an indentation here?

Reply:
idk vscode with latex workshop just automatically indent the file